exercises/bank-account/bank_account.py

```diff
@@ -0,0 +1,21 @@
+
+
+class BankAccount(object):
+
+    def __init__(self):
+        pass
+
+    def getBalance(self):
```

Comment:
Following [PEP8](https://www.python.org/dev/peps/pep-0008/) we use `snake_case` for function and method names, so this should be `get_balance`.